demo/remote/azure/terraform/modules/azure-hashistack/templates/azure_autoscaler.nomad

```diff
@@ -0,0 +1,119 @@
+job "autoscaler" {
+  datacenters = ["dc1"]
+
+  group "autoscaler" {
+    count = 1
+
+    task "autoscaler" {
+      driver = "docker"
+
+      config {
+        image   = "hashicorp/nomad-autoscaler:0.2.0"
```

Comment:
TODO: make sure this is the right version.

Reply:
In order to test this before everything is released, you can get a nightly build from the Autoscaler, place it in `/usr/bin` in the client and use the `exec` driver.